.github/workflows/local-wisconsin.yml

```diff
@@ -0,0 +1,137 @@
+      pss_base_url:
+        description: "URL for Primary Source Sets"
+        required: true
+        default: "https://dp.la/pssapi"
+      local_id: 
+        description: "Key for local"
+        required: true
+        default: "wisconsin"
+      ga_tracking_id: 
+        description: "Analytics tracking ID"
+        required: true
+        default: "UA-128040088-1"
+      omeka_url: 
+        description: "Omeka URL"
+        required: true
+        default: "http://omeka.internal.dp.la"
+
+jobs:
+  ecr:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Read .nvmrc
+        run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
+        id: nvm
+      
+      - name: Use Node.js ${{ steps.nvm.outputs.NODE_VERSION }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
+      
+      - name: Install Yarn
+        run: npm install -g yarn
+
+      - name: Set AWS credentials
+        if: ${{ !env.ACT }}
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+            aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+            aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+            aws-region: us-east-1    
+
+      - name: Login to Amazon ECR
+        if: ${{ !env.ACT }}
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      # Building the frontend app
+      - name: Create and populate constants/env.js
+        run: |
+          touch constants/env.js
+          echo export const SITE_ENV = "${{ github.event.inputs.site_env }}"; >> constants/env.js
+          echo export const USER_BASE_URL = "${{ github.event.inputs.user_base_url }}"; >> constants/env.js
+          echo export const WORDPRESS_URL = "${{ github.event.inputs.wordpress_url }}"; >> constants/env.js
+          echo export const PRO_BASE_URL = "${{ github.event.inputs.pro_base_url }}"; >> constants/env.js
+          echo export const PSS_BASE_URL = "${{ github.event.inputs.pss_base_url }}"; >> constants/env.js
+          echo export const LOCAL_ID = ""${{ github.event.inputs.local_id }}""; >> constants/env.js
+          echo export const gaTrackingId = "${{ github.event.inputs.ga_tracking_id }}"; >> constants/env.js
+        shell: bash
+
+      - name: Add path 
+        run: |
+          echo "HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin" >> $GITHUB_PATH
+        shell: bash
+      
+      - name: Build the app with Yarn
+        env: 
+          SITE_ENV: "${{ github.event.inputs.site_env }}"
+          LOCAL_ID: "${{ github.event.inputs.local_id }}"
+          BASE_URL: "${{ github.event.inputs.base_url }}"
+          OMEKA_URL: "${{ github.event.inputs.omeka_url }}"
+
+        run: |
+          yarn clean
```

Comment:
does there need to be a `yarn` invocation between clean and build to install dependencies?

Reply:
I don't think so, testing locally appears to install all the dependencies. 

```
[1/4] Resolving packages...
[1/4] Resolving packages...
[2/4] Fetching packages...
[2/4] Fetching packages...
info There appears to be trouble with your network connection. Retrying...
[3/4] Linking dependencies...
warning " > babel-loader@8.2.4" has unmet peer dependency "webpack@>=2".
warning " > copy-webpack-plugin@5.1.2" has unmet peer dependency "webpack@^4.0.0 || ^5.0.0".
warning " > html-webpack-plugin@3.2.0" has unmet peer dependency "webpack@^1.0.0 || ^2.0.0 || ^3.0.0 || ^4.0.0".
warning " > react-aria-modal@4.0.1" has incorrect peer dependency "react@^15.0.0 || ^16.0.0".
warning "react-aria-modal > focus-trap-react@8.9.2" has unmet peer dependency "prop-types@^15.8.1".
warning "react-aria-modal > react-displace@2.3.0" has incorrect peer dependency "react@0.14.x || ^15.0.0 || ^16.0.0".
warning "react-aria-modal > react-displace@2.3.0" has incorrect peer dependency "react-dom@0.14.x || ^15.0.0 || ^16.0.0".
warning " > react-markdown@4.3.1" has incorrect peer dependency "react@^15.0.0 || ^16.0.0".
warning " > sw-precache-webpack-plugin@0.11.5" has unmet peer dependency "webpack@^1 || ^2 || ^2.1.0-beta || ^2.2.0-beta || ^3 || ^4".
warning " > file-loader@4.3.0" has unmet peer dependency "webpack@^4.0.0".
warning " > url-loader@2.3.0" has unmet peer dependency "webpack@^4.0.0".
[3/4] Linking dependencies...
warning " > babel-loader@8.2.4" has unmet peer dependency "webpack@>=2".
```